pybossa/view/fileproxy.py

```diff
@@ -92,6 +93,10 @@ def encrypted_file(store, bucket, project_id, path):
     if not signature:
         current_app.logger.exception('Project id {} no signature {}'.format(project_id, path))
         raise Forbidden('No signature')
+    size_signature = len(signature)
+    if size_signature > SIGNATURE_MAX_SIZE:
+        current_app.logger.exception('Project id {}, path {} invalid signature length {}.'.format(project_id, path, size_signature))
```

Comment:
Can we make the error more specific, maybe 
`current_app.logger.exception('Project id {}, path {} invalid signature, signabure length {} exceeds maximim length {}.'.format(project_id, path, size_signature, SIGNATURE_MAX_SIZE))`?

Reply:
ok, will update log message